### PR TITLE
ACTIN-1325: Make country and city nullable in CKB datamodel and mapping

### PR DESCRIPTION
**Pull request by kduyvesteyn**
@cbruel Could you please review if facility curation in `CkbCurator` still works as expected?

### CODE REVIEW

File: algo/src/test/java/com/hartwig/serve/sources/ckb/curation/CkbCuratorTest.java [hunk 1]
```diff
@@ -47,6 +47,12 @@ public void canCurateFacilityNameBasedOnName() {
         assertEquals(expectedLocation, curator.curateFacilityName(location2));
     }
 
+    @Test
+    public void canCurateFacilityWithMissingCity() {
+        Location missingCity = createLocation(null, "facility", null);
+        assertEquals(missingCity, curator.curateFacilityName(missingCity));
+    }
```
Comment (kzuberihmf) [inline algo/src/test/java/com/hartwig/serve/sources/ckb/curation/CkbCuratorTest.java:54]: I suppose country could also be null, if you want to include that in the test.

**Review by kzuberihmf — APPROVED** [review]
LGTM

File: algo/src/test/java/com/hartwig/serve/sources/ckb/curation/CkbCuratorTest.java (hunk 1 above)
Comment (kduyvesteyn) [reply algo/src/test/java/com/hartwig/serve/sources/ckb/curation/CkbCuratorTest.java:54]: good one, added

**Comment by cbruel** [comment]
@kduyvesteyn Looks good!

**Pull request merged**